### PR TITLE
Fix and improve the Japanese localization

### PR DESCRIPTION
**Pull request by shuuji3**
Hi, this PR fixes and improves the Japanese localization.

### CODE REVIEW

File: i18n/ja.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 module.exports = {
-  name: '英語',
```
Comment (shuuji3) [inline i18n/ja.js:2]: This text is shown in the bottom-left language selector. It should be "日本語" (Japanese) not "英語" (English).

File: i18n/ja.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  name: '日本語',
   key: 'ja',
-  'Open in Debug Map': 'デバッグされた地図で開く',
-  'Open in Mapillary': 'Mapilaryで開く',
+  'Open in Debug Map': 'デバッグ地図で開く',
+  'Open in Mapillary': 'Mapillaryで開く',
```
Comment (shuuji3) [inline i18n/ja.js:5]: This is a just typo.

**Review by mjjbell — APPROVED** [review]
LGTM, thanks!

**Comment by shuuji3** [comment]
Thank you for merging! 🙂

**Pull request merged**